lib/record_store/zone/config.rb

```diff
@@ -24,7 +27,7 @@ def supports_alias?
       def to_hash
         config_hash = {
           providers: providers,
-          ignore_patterns: ignore_patterns,
+          ignore_patterns: ignore_patterns.map {|ignore_pattern| ignore_pattern.to_hash },
```

Comment:
You can use `ignore_patterns.map(&:to_hash)` and that will have the same result.